homeassistant/helpers/config_validation.py

```diff
@@ -106,6 +107,22 @@
 
 TIME_PERIOD_ERROR = "offset {} should be format 'HH:MM', 'HH:MM:SS' or 'HH:MM:SS.F'"
 
+
+class UrlProtocolSchema(StrEnum):
+    """Valid URL protocol schema values."""
+
+    HTTP = "http"
+    HTTPS = "https"
+    HOMEASSISANT = "homeassistant"
+
+
+EXTERNAL_URL_PROTOCOL_SCHEMA_LIST = frozenset(
+    {UrlProtocolSchema.HTTP, UrlProtocolSchema.HTTPS}
+)
+COMBINED_URL_PROTOCOL_SCHEMA_LIST = frozenset(
```

Comment:
Should this be named `CONFIGURATION_URL_PROTOCOL_SCHEMA_LIST` to match the name of the validator?